src/neo/models/neo4j/actor.js

```diff
@@ -0,0 +1,10 @@
+const _ = require('lodash');
```

Comment:
All of the logic in this file is the same. You can just rename the model when you import it and rename it there, unless there is a reason for each model having this separate file.